HW4/task4.py

```diff
@@ -0,0 +1,14 @@
+# 4. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти якого типу введені дані.
+# Використати match, case і вбудовані функції Python
+
+text = True
```

Comment:
nit: it could be solved like in the previous task
```suggestion
text = input('Please, type something: ')
```